stripes/public/js/combobox.bundle.js

```diff
@@ -1,7 +1,7 @@
 import Fuse from 'fuse.js';
 
 class Combobox {
-	constructor(wrapper, on_change) {
+	constructor(wrapper, on_change, fetching_data) {
```

Comment:
No need for a separate event `fetching_data`... You already have `on_change` .. just run the fetching code in the `on_change` handler